File: Python/D20-W/6/6.py
# Write a python program to multiply all the numbers stored in a list.

num_list = []

n = int(input("Enter a range for list: "))

for i in range(0, n):
    num = int(input("Enter number : "))
    num_list.append(num)

total = 1
for i in num_list:
    total = total * i

print(num_list)
print(total)

File: C/D4/6/6.cpp
// Write a C program to for below output.
// 1
// 2 2
// 3 3 3

#include <stdio.h>

int main(){
    int i,j;

    for(i = 1; i <= 3; i++){
        for(int j = 1; j <= i; j++){
            printf("%d ", i);
        }
        printf("\n");      
    }
}